README.md

```diff
@@ -1,19 +1,99 @@
 # KNoT Fog connector service
 
-This is a KNoT gateway service that connects the fog with a cloud service.
+This is a KNoT Gateway service that connects the fog to a cloud service.
 
 ## Supported services
 
-* [KNoT Cloud](https://github.com/CESARBR/knot-fog-connector-knot-cloud)
+* [KNoT Cloud](https://github.com/CESARBR/knot-fog-connector-knot-cloud) (under development)
```

Comment:
If both `knot-cloud` and `fiware` have stable versions, there is no need to add *(under development)* at the end of line.

Reply:
Done! :smile: